### ISSUE TITLE
Hide Statusbar

### ISSUE THREAD

**Issue by MggMuggins** [pithos/pithos#487]
Alright, so I did a quick search through issues, and found #81. It didn't have much about the statusbar, and I didn't read through the whole thing, but I didn't get much on this.

Basically I'm wondering what would be required to hide the statusbar when it's not in use. For example, when pithos is "Getting Songs", the statusbar would be shown, but just about any other time it's empty and eating up space. I'm not familiar with Gtk, or its documentation. What would be required to do this?

**Comment by JasonLG1979** [comment]
I agree. The statusbar is pretty useless. It will probably go away in a future UI redesign. I played with removing it before the redesign but visually it make the UI look very top heavy without it.

No time table on the redesign(1.5 or 1.6 maybe?), there are a few things that I would to get done before that but here is a Glade preview of a very rough non-functional work in progress that I've been playing with. Forgive the double title bar and button states and what not.

![screenshot from 2017-09-04 07-13-31](https://user-images.githubusercontent.com/6667703/30038037-6fe40032-9186-11e7-9261-d72fee10aca1.png)
